assets/airports/katl.json

```diff
@@ -943,66 +914,133 @@
-                ["EAGYL", "SHELE", "SUMNR"],
-                ["CHOPI", "ONYON", "GAASS"],
-                ["RAIIN", "TRAPE", "SNEVE"],
-                ["ONYON", "FRYES"],
-                ["DDAVS", "TABBL"],
-                ["TRAYE", "TABBL"],
-                ["MMMOE", "YORRR"]
+                []
             ]
         },
-        "DRMMM1": {
-            "icao": "DRMMM1",
-            "name": "Drum One",
+        "JJEDI2": {
+            "icao": "JJEDI2",
+            "name": "Jedi Two",
             "entryPoints": {
-                "EEZRA": ["EEZRA", ["JCKEE", "A240+"], ["OWNNS", "A170+|S280"]],
-                "KNGGG": ["KNGGG", "THRGD", ["ROSAA", "A240+"], ["LUWIS", "A170+|S280"]]
+                "EEWOK": ["EEWOK", "LAYUH", ["FLKNN", "A140+|S280"], ["CHWEE", "A110+|S250"]],
+                "HOTHH": ["HOTHH", "LLNDO", ["XWNNG", "A140+|S280"], ["BBFET", "A110+|S250"]],
+                "LARZZ": [["LARZZ", "A240+"], "NEWHP", ["DTSTR", "A140+|S280"], ["WOKIE", "A110+|S250"]],
+                "LAYUH": ["LAYUH", ["FLKNN", "A140+|S280"], ["CHWEE", "A110+|S250"]],
+                "MELNM": ["MELNM", "LAYUH", ["FLKNN", "A140+|S280"], ["CHWEE", "A110+|S250"]],
+                "SKWKR": ["SKWKR", "LAYUH", ["FLKNN", "A140+|S280"], ["CHWEE", "A110+|S250"]],
+                "TYFTR": ["TYFTR", "NEWHP", ["DTSTR", "A140+|S280"], ["WOKIE", "A110+|S250"]]
             },
-            "body": [["DRMMM", "A130+|S280"], "MRCHH"],
+            "body": [["JJEDI", "A100+|S250"], ["DAFII", "A80"], "GRHAM", "^POOBA", "#004"],
             "rwy": {
-                "8L": [["SWARD", "A90|S250"]],
-                "8R": [["SWARD", "A90|S250"]],
-                "9L": [["CHNCE", "A100|S250"], ["NAVVY", "A80+"], ["RYENN", "A70+"], ["AAKAY", "A60+"]],
-                "9R": [["CHNCE", "A100|S250"], ["NAVVY", "A80+"], ["ANDIY", "A70+"], ["DFINS", "A60+"]],
-                "10": [["CHNCE", "A100|S250"], "TOMKT", "#178"],
-                "26L": [["HIDRO", "A130|S280"], "RAIIN", "KLOWD", "POORS", "KYMMY", "KEAVY"],
-                "26R": [["HIDRO", "A130|S280"], "RAIIN", "KLOWD", "POORS", "KYMMY", "KEAVY"],
-                "27L": [["HIDRO", "A130|S280"], "EAGYL", "SHURT", "SWETE", "KRRIS", "TAAMY"],
-                "27R": [["HIDRO", "A130|S280"], "EAGYL", "SHURT", "SWETE", "KRRIS", "TAAMY"],
-                "28": [["HIDRO", "A130|S280"], "EAGYL", "SHURT", "SWETE", "KRRIS", "TAAMY"]
+                "KATL26L": [],
+                "KATL26R": [],
+                "KATL27L": [],
+                "KATL27R": [],
+                "KATL28": []
             },
             "draw": [
-                ["EEZRA", "JCKEE", "OWNNS", "DRMMM", "MRCHH", "CHNCE", "TOMKT"],
-                ["KNGGG", "THRGD", "ROSAA", "LUWIS", "DRMMM"],
-                ["CHNCE", "NAVVY", "ANDIY", "DFINS"],
-                ["MRCHH", "HIDRO", "RAIIN"],
-                ["HIDRO", "EAGYL"],
-                ["RYENN", "AAKAY"]
+                []

```

Comment:
Why are we not drawing here? Would be nice for future plans, if STARs are drawn.